cmd/state/internal/cmdtree/pull.go

```diff
@@ -18,21 +18,16 @@ func newPullCommand(prime *primer.Values) *captain.Command {
 		locale.Tl("pull_description", "Pull in the latest version of your project from the ActiveState Platform"),
 		prime,
 		[]*captain.Flag{
-			{
-				Name:        "force",
-				Shorthand:   "",
-				Description: locale.Tl("flag_state_pull_force_description", "Force pulling specified project even if it is unrelated to checked out one"),
-				Value:       &params.Force,
-			},
 			{
 				Name:        "set-project",
 				Shorthand:   "",
-				Description: locale.Tl("flag_state_pull_set_project_description", "project even if it is unrelated to checked out one"),
+				Description: locale.Tl("flag_state_pull_set_project_description", "Pull from the specified project instead of the checked out one"),
 				Value:       &params.SetProject,
 			},
 		},
 		[]*captain.Argument{},
 		func(cmd *captain.Command, args []string) error {
+			params.Force = globals.NonInteractive
```

Comment:
nit: If this doesn't need to happen within the handler itself, it is probably more appropriate to set this in the declaration at line 13.

Reply:
oof, nm, I didn't think I was late enough on this review to check if it was already completed. Sorry about that.

Reply:
I had it there, but it failed the unit tests because I think the globals contents are edited after declaration.